client/trackpack/src/App.js

```diff
@@ -6,33 +6,49 @@ import { ChakraProvider } from "@chakra-ui/react";
 import Topnav from "./components/Topnav";
 import Signup from "./views/signup/signup";
 import Signin from "./views/signin/signin";
-import Homepage from "./components/homepage";
+import Package from "./components/package";
+import Packages from "./views/tracking/packages";
+import Homepage from "./views/home/homepage";
 import "bootstrap/dist/css/bootstrap.min.css";
 
 function App() {
-  return (  
+  return (
     <ChakraProvider>
-   <Router>
-      <div>
-          <Topnav/>
+      <Router>
+        <div>
+          <Topnav />
           <Switch>
             <Route exact path="/">
-              <Homepage title="Homepage"/>
+              <Homepage title="Homepage" />
             </Route>
             <Route path="/signup">
-              <Signup/>
+              <Signup />
             </Route>
             <Route path="/signin">
-              <Signin/>
+              <Signin />
+            </Route>
+            <Route path="/packages">
+              <Packages>
+                <Package
```

Comment:
I know this is hard-coded in order to test the Package component but this should be in its own page.

Reply:
What do you mean? I thought all of our views and components are routed using App.js.

Reply:
Do you mean putting package component inside the packages view instead like this?
![image](https://user-images.githubusercontent.com/25624481/112926376-5ac78a80-90e1-11eb-8162-178920b70f81.png)

Reply:
yeah. The hard coded components should be where you think the final component logic will be, in order to prevent confusion.